### PR TITLE
Vladis Urbanski hw5task4 review please

### PR DESCRIPTION
**Pull request by vladis1350**

### CODE REVIEW

**Review by TLittleProgrammer — APPROVED** [review]
lgfm

File: homework/Vladislav Urbanski/less-1 edit test/ex4_Stock/src/com/vladis1350/App.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package com.vladis1350;
+
+import com.vladis1350.test.StokTest;
+
+public class App {
+    public static void main(String[] args) throws Exception {
+        StokTest stokTest = new StokTest();
+        stokTest.testShouldVerifyNameCompany();
+        stokTest.testShouldInfoCompanyAfterOneUpdate();
```
Comment (javaGuruBY) [inline homework/Vladislav Urbanski/less-1 edit test/ex4_Stock/src/com/vladis1350/App.java:9]: test should WAT?

File: homework/Vladislav Urbanski/less-1 edit test/ex5_UserLoginService/src/com/vladis1350/bean/User.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+    public void blockUser() {
+        if (userAccess && getCountAttempt() == 0) {
+            setUserAccess(false);
+        } else {
+            setUserAccess(true);
+        }
+    }
+
+    public boolean isUserAccess() {
```
Comment (javaGuruBY) [inline homework/Vladislav Urbanski/less-1 edit test/ex5_UserLoginService/src/com/vladis1350/bean/User.java:30]: isUserHasAcces ?

File: homework/Vladislav Urbanski/less-1 edit test/ex5_UserLoginService/src/com/vladis1350/service/UserLoginService.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
+package com.vladis1350.service;
```
Comment (javaGuruBY) [inline homework/Vladislav Urbanski/less-1 edit test/ex5_UserLoginService/src/com/vladis1350/service/UserLoginService.java:1]: Did I showed you how to wrote login service? 
Did I asked to re-write this?

**Review by javaGuruBY — CHANGES_REQUESTED** [review]
boolean login(User user, String password) {
  isEnteredPasswordEqualWithUserPassword(user, password) {
    dropAccessTriesCounter();
    return true;   
} else {
    decreaseAccessTriesCounter();
    isUserShouldBlocked();
}


пошарь коллегам

**Pull request closed without merging**